trunk/src/graphics/paraview/vector2vtk.F90

```diff
@@ -215,7 +216,20 @@ subroutine vector2vtk(Sname,Sfile,Snick,Idx, Ic)
 !
 !..Step 4 : Write to file with HDF5
 !
-   if (RANK .eq. ROOT) call attr_write(Sname,len(Sname),Sfile,len(Sfile),Snick,len(Snick))
+   if (RANK .eq. ROOT) then
+      if(VIS_VTU .eqv. .false.) then
```

Comment:
Style
```suggestion
      if (.not. VIS_VTU) then
```

Reply:
changes implemented: throughout the new commit